### ISSUE TITLE
Improve the error message of partial methods

### ISSUE THREAD

**Issue by qqfunc** [beeware/rubicon-objc#461]
### What is the problem or limitation you are having?

The current error message is as follows:

```python
>>> obj.someMethod(..., invalidArg=...)
```
```
Traceback (most recent call last):
    ...
ValueError: No method was found starting with 'someMethod' and with keywords {'invalidArg'}
Known keywords are:
frozenset({''})
frozenset({'', 'validArg'})
```

This will be as follows after #453 is resolved because `frozenset` will be changed to `tuple`:

```
ValueError: No method was found starting with 'someMethod' and with keywords ('invalidArg')
Known keywords are:
('')
('', 'validArg')
```

### Describe the solution you'd like

I think the converted selector name and available selector names should be shown.

```
ValueError: Invalid selector 'someMethod:invalidArg:'. Available selectors are: 'someMethod:', 'someMethod:validArg:'
```

### Describe alternatives you've considered

It is also possible to show descriptions of Python, not selectors.

```
ValueError: Invalid method 'someMethod(..., invalidArg=...)'. Available methods are: 'someMethod(...)', 'someMethod(..., validArg=...)'
```

Actual values of arguments are represented as `...`.

**Comment by freakboy3742** [comment]
Agreed this would be a good improvement - especially if it were paired with an implementation of #453 and #148. 

Of the two syntax options you've presented, I'd lean towards the "selector" description - any user is trying to invoke an ObjC method, so I don't think we gain anything by trying to avoid ObjC syntax in error messages.

**Comment by qqfunc** [comment]
Thanks for agreeing. I will try it with the issues.

**Comment by freakboy3742** [comment]
Fixed by #462.
